src/datatypes.py

```diff
@@ -1,8 +1,11 @@
 x = 5
-y = "7"
+y = 7
```

Comment:
The challenge here is to leave these definitions for `x` and `y` as they are and change the types when you combine them. (Hint: Python has built-in functions to turn things into `str`ings and `int`egers 👍)

Reply:
Hi, Ellen! 
This is well noted. Thanks for the tip. :)